Gopkg.toml

```diff
@@ -5,3 +5,7 @@
 [[constraint]]
   name = "github.com/confluentinc/confluent-kafka-go"
   revision = "767701354ee866ea3a63110035df26a8cccc3dee"
+
+[[constraint]]
+  name = "github.com/kelseyhightower/envconfig"
+  revision = "1f94cfab3dcc47159a6dcb83588baa3d8260e78a"
```

Comment:
I don't see this dependency being imported anywhere?

Reply:
Not in this PR, these are some fixes I did while I was integrating the library in some of the processors.

The dependency is used here:

https://github.com/blendle/go-streamprocessor/pull/52

Locally everything worked as expected, since I already `go get`ed the dependency, and the CI unfortunately doesn't run yet, since this code isn't on master yet, but once I added this library to another project, this error showed up when I tried to install the dependencies inside a container.